### ISSUE TITLE
Problems with the rendering SimpleMDE component

### ISSUE THREAD

**Issue by RIP21** [benrlodge/react-simplemde-editor#24]
Hi there. 
I have an issue with the rendering. And it cause by wrong building of your module or something like that.
Once I'm import or require your component this way:
`import SimpleMDE from 'react-simplemde-editor';`
And then add it to the render method, the following problem appear.
`Uncaught Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. Check the render method of EditPostForm.`

`Warning: React.createElement: type should not be null, undefined, boolean, or number. It should be a string (for DOM elements) or a ReactClass (for composite components). Check the render method of EditPostForm.`

It figured later that It can be fixed importing module this way
`import SimpleMDE from 'react-simplemde-editor/src';`
It start to work fine, but also start to log tons of shit to the webpack console looks like that (last 20 rows of that)
```

  [920] ./~/react-simplemde-editor/~/react/lib/ReactDefaultPerfAnalysis.js 5.79 kB {0} [built]
  [921] ./~/react-simplemde-editor/~/fbjs/lib/performanceNow.js 844 bytes {0} [built]
  [922] ./~/react-simplemde-editor/~/fbjs/lib/performance.js 612 bytes {0} [built]
  [923] ./~/react-simplemde-editor/~/react/lib/ReactVersion.js 379 bytes {0} [built]
  [924] ./~/react-simplemde-editor/~/react/lib/renderSubtreeIntoContainer.js 463 bytes {0} [built]
  [925] ./~/react-simplemde-editor/~/react/lib/ReactDOMServer.js 766 bytes {0} [built]
  [926] ./~/react-simplemde-editor/~/react/lib/ReactServerRendering.js 3.3 kB {0} [built]
  [927] ./~/react-simplemde-editor/~/react/lib/ReactServerBatchingStrategy.js 673 bytes {0} [built]
  [928] ./~/react-simplemde-editor/~/react/lib/ReactServerRenderingTransaction.js 2.3 kB {0} [built]
  [929] ./~/react-simplemde-editor/~/react/lib/ReactIsomorphic.js 2.05 kB {0} [built]
  [930] ./~/react-simplemde-editor/~/react/lib/ReactDOMFactories.js 3.36 kB {0} [built]
  [931] ./~/react-simplemde-editor/~/react/lib/ReactElementValidator.js 10.8 kB {0} [built]
  [932] ./~/react-simplemde-editor/~/fbjs/lib/mapObject.js 1.47 kB {0} [built]
  [933] ./~/react-simplemde-editor/~/react/lib/onlyChild.js 1.21 kB {0} [built]
  [934] ./~/react-simplemde-editor/~/react/lib/deprecated.js 1.77 kB {0} [built]
  [935] ./~/simplemde/dist/simplemde.min.js 269 kB {0} [built]
  [936] ./~/react-simplemde-editor/src/services/idGenerator.js 98 bytes {0} [built]
  [937] ./~/react-simplemde-editor/src/utils/noop.js 32 bytes {0} [built]

WARNING in ./~/simplemde/dist/simplemde.min.js
Critical dependencies:
7:417-424 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
 @ ./~/simplemde/dist/simplemde.min.js 7:417-424
```

For me (I'm very new in JS and webpack etc.) It looks like a problem with building process of the package. Guys from one of the local JS chats who helped me figure that out saying that you include to the build all React itself or something like that.
Anyway, I like your component very much, but please fix it :)
My particular use case you can find out at this repo.
https://github.com/RIP21/ololos-blog
`components/post/EditPostForm.js `

**Comment by RIP21** [comment]
Also, I don't know why, but after update of Webpack to version 1.13.2 it also start to broke minifying process :) Arguing about string wrapped by ` symbol. After all these problems I just extract your component implementation fix this string concatenation using ` and add all dependencies it need, and removed 'react-simplemde-editor'. 
So it finally minify and working properly. But still argue about 
`WARNING in ./~/simplemde/dist/simplemde.min.js
Critical dependencies:
7:417-424 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
 @ ./~/simplemde/dist/simplemde.min.js 7:417-424`

**Comment by benrlodge** [comment]
Thanks for bringing this up - the library was recently updated to use the output in /dist, but I failed to ensure it was exported as umd. This should be fixed with version `3.6.4`

**Comment by RIP21** [comment]
Thanks a lot. Now it work fine! :) And pretty clean output in Webpack. Thanks a lot! :)
